src/main/kotlin/no/nav/familie/klage/brev/BrevService.kt

```diff
@@ -151,9 +162,25 @@ class BrevService(
                         klageMottatt = klageMottatt,
                     )
                 } else {
-                    fun getOrThrow(verdi: String?, felt: String) = verdi
+                    fun getOrThrow(
+                        verdi: String?,
+                        felt: String,
+                    ) = verdi
                         ?: throw Feil("Behandling med resultat $behandlingResultat mangler $felt for generering av brev")
 
+                    val klagefristUnntakOppfylt =
+                        formkrav.klagefristOverholdtUnntak in
+                            listOf(FormkravFristUnntak.UNNTAK_SÆRLIG_GRUNN, FormkravFristUnntak.UNNTAK_KAN_IKKE_LASTES)
+
+                    if (klagefristUnntakOppfylt &&
+                        formkrav.brevtekst == null
+                    ) {
+                        throw ApiFeil(
+                            "Fritekst må fylles ut.",
+                            httpStatus = HttpStatus.BAD_REQUEST,
+                        )
+                    }
```

Comment:
Her kunne du eventuelt brukt funksjonen `brukerfeilHvis`, som kaster en `ApiFeil` med status `BAD_REQUEST`

```kotlin
brukerfeilHvis(klagefristUnntakOppfylt && formkrav.brevtekst == null) {
    "Fritekst må fylles ut."
}
```

Fortjener denne en bedre feilmelding? "Hvis unntak for klagefrist er oppfylt, må begrunnelse fylles ut i fritekstfelt" eller noe sånt? 🤔

Reply:
Ja, godt poeng!